tools/npz2rds/npz2rds.sh

```diff
@@ -80,7 +85,16 @@ if [ -f $outputfile ]; then
 fi
 
 ## Make tmp directory
-tmpdir=`mktemp -d`
+if [ -z "$tmpdir" ]
```

Comment:
Looks like the convention in this file is to put the `then` on the same line, e.g. 
```suggestion
if [ -z "$tmpdir" ]; then
```